### ISSUE TITLE
@module reports "Tag @module must have a type" after upgrade to 15.9.0

### ISSUE THREAD

**Issue by doberkofler** [gajus/eslint-plugin-jsdoc#375]
After upgrading from 15.8.4 to 15.9.0 a JSDOC like this

```typescript
/**
*	A lightweight, customizable javascript timepicker inspired by Google Calendar.
*	@module timepicker
*/
```
reports the error `Tag @module must have a type (jsdoc/valid-types)`  but according to the [JSDOC](https://jsdoc.app/tags-module.html) documentation the type is optional.

**Comment by brettz9** [comment]
Confirmed. `enum` and `member`/`var` should are optional types as well. Hope to have a fix out shortly.

**Comment by doberkofler** [comment]
@brettz9 That's quick! Thank you!
